### ISSUE TITLE
On the line of how division of labour is related to trade, where does specialisation fit in here? Is specialistion the same as division of labour?

### ISSUE THREAD

**Issue by Naty-fedorova** [abjeffre/market_integration#1]

**Comment by abjeffre** [comment]
Yea that is a good question. 
My intuition is to define them at different scales:
Specialization for individuals.
DOL for groups and populations. 

Specialization is a trait of individuals and it refers to the number of different tasks they preform and the amount of time(effort) they spend doing it.  We can define it using an entropy measure

S = sum(pi log(pi))

Where pi is the share (read proportion) of time spent on task i.
This definition has a few properties to it that are nice, but first note that higher values indicate more specialization. 
First, it is decreases with the more tasks you add.
Second, it decreases with the greater share each task holds.
Lastly, it is additive.   

This, i am fairly certain is an optimal definition of specialization now the question is this.  IS DOL just the aggregate mean of the  specialization measure summed across all individuals in a population.   What are your thoughts?

**Comment by Naty-fedorova** [comment]
Ok, that makes sense to me too.

I don't quite understand the equation - where is the term relating to the different tasks they perform? Otherwise it seems good, I'm also wondering whether it doesn't need some sort of learning (or broadly cost of pursuing) term, but maybe that will already be captured in time spent. 

I have no idea as to the aggregate mean of specialization being the degree of DOL. I feel like DOL captures how some whole chunk of labour that needs to be done is divided by all the individuals doing it, but then since there are sure to be efficiency gains through specialization, it somehow changes that whole chunk that needs to be done.

**Comment by Naty-fedorova** [comment]
Also - under this definition, does it imply you cannot have a division of labour in a group in which you do not have specialists?

**Comment by abjeffre** [comment]
pi is really p with a subscript i.  I just cant be bothered to figure it out on here just yet.  And so you sum from i=1 to i=n, where n is the maximum number of tasks they preform. 
p then is a share share to total time.  

For example, say an invividual spends 5 days a month farming , 15 fishing and 10 running a small shop.  
p1 = 1/6 = share of time spent farming.
p2 = 1/2 = share of time spent fishing.
p3 = 1/3 = share of time spent with a business.

we would then apply pi*log(pi) to p1, p2, and p3, and sum the values together.   This then gives us a measure of specialization.  Note that the maximum value is zero ln(1) = 0 and everything else is less than that.

**Comment by abjeffre** [comment]
> Also - under this definition, does it imply you cannot have a division of labour in a group in which you do not have specialists?

No, this definition is flexible because it is continuous.  It says that each individual is a specialist to a certain degree and measures that.  

I just realized that if we simply take the aggregate sum of this then we DO NOT MEASURE DOL. For example if  each person  preforms the same single task as everyone else (ie. everyone farms and only farms, p1 =1) this measure would assume that the DOL _is the same_ as if every individual preformed a single task that is different from everyone else (ie. everyone has a unique occupation and preforms only that, p1= 1 AS WELL). 

I will think on how to merge these metrics into a single value.

**Comment by abjeffre** [comment]
On a coarse level, the same entropy measure can be used to measure DOL in the population.  You just say there are n occupations in a population and calculate the entropy based off of the number of individuals who have that occupation.   The problem is that doing it that way does not provide a link between individual specialization and DOL, because it assumes that each individual does one, and only one task. (which is likley the case for historical populations and the modern poor)
